Use golang.org/x/sys instead of syscall?

While installing transocks on i686 system, I got the following error:
```
go/src/github.com/cybozu-go/transocks/original_dst_linux.go:14:3: undefined: syscall.SYS_GETSOCKOPT
```

[The official document](https://golang.org/pkg/syscall/) says:
> Deprecated: this package is locked down. Callers should use the corresponding package in the golang.org/x/sys repository instead. That is also where updates required by new systems or versions should be applied. See https://golang.org/s/go1.4-syscall for more information.

I manually replaced syscall with golang.org/x/sys/unix. It worked well.

For better OS support, use `golang.org/x/sys` instead?

--- a/go.mod
+++ b/go.mod
@@ -6,4 +6,5 @@ require (
 	github.com/cybozu-go/netutil v1.2.0
 	github.com/cybozu-go/well v1.8.1
 	golang.org/x/net v0.0.0-20180911220305-26e67e76b6c3
+	golang.org/x/sys v0.0.0-20180906133057-8cf3aee42992
 )

--- a/original_dst_linux.go
+++ b/original_dst_linux.go
@@ -3,9 +3,9 @@
 package transocks
 
 import (
+	syscall "golang.org/x/sys/unix"
 	"net"
 	"os"
-	"syscall"
 	"unsafe"
 )
 

